packages/api-plugin-fulfillment-method-pickup-store/src/startup.js

```diff
@@ -0,0 +1,21 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+import checkAndCreateFulfillmentMethod from "./util/checkAndCreateFulfillmentMethod.js";
+/**
+ * @summary Called on startup to create the root entry of this fulfillment type in Fulfillment collection
+ * @param {Object} context Startup context
+ * @returns {undefined}
+ */
+export default async function fulfillmentMethodPickupStoreStartup(context) {
+  context.appEvents.on("afterShopCreate", async (payload) => {
+    const { shop } = payload;
+    const shopId = shop._id;
+
+    // We do not have validatePermissions in context during this startup stage, hence commenting below
+    // await context.validatePermissions("reaction:legacy:fulfillmentTypes", "read", { shopId });
```

Comment:
Why do we have this comment left here if we are not going to validate the permissions?